### PR TITLE
Doc: Add ENVIRONMENT_VARIABLES.md to docs folder

### LINKED ISSUE #92: Doc: Add ENVIRONMENT_VARIABLES.md
## Description
As a student,
I need add ENVIRONMENT_VARIABLES.md,
so that I can make sure contributors can setup BIT backend correctly.

## Mocks
NA

### Update [Required]
- [ ] add ENVIRONVEMT_VARIABLES.md to the `doc` folder

### Enhancement to Update [Optional]
NA

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hour

### PR DESCRIPTION
**Pull request by mtreacy002**
### Description
Add ENVIRONMENT_VARIABLES.md to docs folder to help guide contributor setup development environment.

Fixes #92 

### Type of Change:
- Documentation

**Code/Quality Assurance Only**
- This change requires a documentation update (add link to this doc on BIT backend setup environment wiki page)

### How Has This Been Tested?
View the proposed doc on fork repo web browser

### Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by mtreacy002** [comment]
Update @isabelcosta and @anitab-org/bridgeintech-maintainers . This PR is needed to complete the setup instructions on bot BIT backend wiki and frontend readme. Can you please approve? Ta

File: docs/ENVIRONMENT_VARIABLES.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+```
+
+## Environment Variables Description
+
+### Run Configuration
+
+| Environment Variable     | Description                                                                                                                                                                                                                    | Example |
+|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+
+| FLASK_ENVIRONMENT_CONFIG | Short running environment name so that Flask know which configuration to load. Currently, there are 4 options for this: `dev`, `test`, `local` and `prod`.  If you want to use a simple local database (e.g.: sqlite) then set your environment config value as `local`. To use the development environment with a remote database configuration you should use `dev` as a value.  | local     |
```
Comment (isabelcosta) [inline docs/ENVIRONMENT_VARIABLES.md:23]: image

This is not being rendered as a markdown table, can you try to fix this @mtreacy002 ?

**Review by isabelcosta — CHANGES_REQUESTED** [review]

File: docs/ENVIRONMENT_VARIABLES.md (hunk 1 above)
Comment (mtreacy002) [reply docs/ENVIRONMENT_VARIABLES.md:23]: Ok, will do that now.

Comment (mtreacy002) [reply docs/ENVIRONMENT_VARIABLES.md:23]: @isabelcosta . I've fixed thee table. Can you please check if it's ok now? Ta
Screen Shot 2020-07-20 at 11 17 27 am

**Review by isabelcosta — APPROVED** [review]
Looks good 👏 @mtreacy002

**Comment by mtreacy002** [comment]
> Looks good 👏 @mtreacy002

Thanks, @isabelcosta. Btw, since I copy pasted that file from MS backend, someone on MS side need to do the same as the table would also not being rendered properly on that part. Just want to let you know 😉

**Review by foongminwong — APPROVED** [review]

**Pull request merged**